src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -216,15 +226,47 @@ public void biteEntity(int entityId) {
             return;
         }
         if (e != null && e instanceof EntityLivingBase) {
-
             if (e.getDistance(player) <= player.getEntityAttribute(EntityPlayer.REACH_DISTANCE).getAttributeValue() + 1) {
-                biteEntity((EntityLivingBase) e);
+                if (determineBiteType(((EntityLivingBase) e)) == BITE_TYPE.ATTACK) {
+                    biteEntity(((EntityLivingBase) e));
+                } else {
+                    victim = e.getEntityId();
+                    PotionEffect effect = new PotionEffect(Potion.getPotionFromResourceLocation("slowness"), 20, 7);
```

Comment:
We could use a custom potion effect here too, this could be used if a player is bitten to render their screen red

Reply:
yeah the render stuff is killing me.  I looked into a potion effect but I wasn't sure how I could do client side rendering stuff on a potion effect.

Reply:
it is not that important, can always add it later if we want